getting_started/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -223,11 +223,10 @@ Other differences:
 GDScript          C#
 ================  ==================================================================
 Color8            Color.Color8
-is_inf            float.IsInfinity
-is_nan            float.IsNaN
```

Comment:
Why remove this?

Reply:
Mathf have them now

Reply:
This shouldn't be in Mathf. What PR added it?